app/controllers/course_templates_controller.rb

```diff
@@ -0,0 +1,89 @@
+class CourseTemplatesController < ApplicationController
+  before_action :set_course_template, except: [:index, :new, :create, :list_for_teachers]
+
+  def index
+    authorize! :read, CourseTemplate
+    ordering = 'LOWER(name)'
+    add_breadcrumb 'Course templates', course_templates_path
+    @course_templates = CourseTemplate.not_hidden.not_dummy.order(ordering)
+    @hidden_course_templates = CourseTemplate.hidden.not_dummy.order(ordering)
```

Comment:
Should we change the `not_hidden` and `hidden` to exclude `dummy` templates by default to remove the repetition of `not_dummy`